### PR TITLE
Create base README

### PR DESCRIPTION
**Pull request by nobody5050**
Using a placeholder image until the robot has competed / been revealed elsewhere. It’s a blacked out version of the CAD, and been approved by Logan.

This readme still needs a bunch added, for instance state/commands should be documented once they are more stable. However, that is outside the scope of this branch and I believe it should be merged in the interim.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,161 @@
...
+|     Purpose/Name     | CAN ID | Motor/Driver Type |
+| :------------------: | :----: | :---------------: |
+|   Front Right Drive  |   2    |     SparkMax      |
+|   Rear Right Drive   |   4    |     SparkMax      |
+|   Rear Left Drive    |   6    |     SparkMax      |
+|   Front Left Drive   |   8    |     SparkMax      |
+|   Front Right Turn   |   1    |     SparkMax      |
+|   Rear Right Turn    |   3    |     SparkMax      |
+|   Rear Left Turn     |   5    |     SparkMax      |
+|   Front Left Turn    |   7    |     SparkMax      |
```
Comment (having11) [inline README.md:83]: Can we add the other motors now too?

File: README.md [hunk 2]
```diff
@@ -1,22 +1,161 @@
...
+
+Before a competition, create a new branch based on `main` and lock it via a GitHub protection rule to prevent pushes. Only use this branch during a competition to avoid unnecessary or breaking deployments to the RIO.
+
+:warning: ***Changes should only be made to this branch if there is a blocking error that must be fixed immediately!*** :warning:
+
+## Subsystems
+
+Following the WPILib command based structure we have broken our robot up into a number of subsystems. They are listed below:
+
+| Subsystem                                                             | Purpose                                                  |
```
Comment (having11) [inline README.md:63]: Nice addition

File: robot.png
Comment (having11) [inline robot.png:1]: CUBE

File: README.md (hunk 1 above)
Comment (nobody5050) [reply README.md:83]: Saving that for a future branch as any IDs beyond what’s currently documented are subject to change

**Review by having11 — APPROVED** [review]

**Pull request merged**